src/Player.cpp

```diff
@@ -3,16 +3,22 @@
 Player::Player()
 {
 	quackBuffer.loadFromFile("Res/quack.wav");
+
 	quackSound.setBuffer(quackBuffer);
 	quackSound.setVolume(15.f);
+
 	duckyTexture.loadFromFile("Res/JanitorDuck.png");
 	duckyTexture.setSmooth(true);
+
+	sf::Vector2u V = duckyTexture.getSize();
```

Comment:
You could add division operator overload do DuckyMath, and then you could just do
```
duckySprite.setOrigin(duckyTexture.getSize() / 2);
```